sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql

```diff
@@ -0,0 +1,41 @@
+WITH campaigns AS (
+  SELECT DISTINCT
+    id,
+    name,
+    FORMAT("%s (%s)", name, CAST(id AS string)) AS felix_compatible_campaign_name
```

Comment:
Yes, I hate this too

Reply:
huh, yeah, those parens make this obnoxious